#
# old_revision [1edea7e71d8eeae001fd5a4c90f75cf6bbdb11d3]
#
# patch "libpurple/protocols/msn/msn.h"
#  from [19cfb512bd575bc1896695498c3ccf995fb680ac]
#    to [32a3ad8dd096af08e72946b96692b199c2d98e54]
#
============================================================
--- libpurple/protocols/msn/msn.h	19cfb512bd575bc1896695498c3ccf995fb680ac
+++ libpurple/protocols/msn/msn.h	32a3ad8dd096af08e72946b96692b199c2d98e54
@@ -83,9 +83,9 @@ typedef enum
 #define MSN_SERVER "messenger.hotmail.com"
 #define MSN_HTTPCONN_SERVER "gateway.messenger.hotmail.com"
 #define MSN_PORT 1863
-#define WLM_PROT_VER		16
+#define WLM_PROT_VER		15
 
-#define WLM_MAX_PROTOCOL	16
+#define WLM_MAX_PROTOCOL	15
 #define WLM_MIN_PROTOCOL	15
 
 #define MSN_TYPING_RECV_TIMEOUT 6
